library_builders.bat

```diff
@@ -193,7 +242,8 @@ call :install_buildessentials
 call :build_zlib
 :: build_bzip2
 :: build_szip
-call :build_blosc
 call :build_zfp
+call :build_blosc
```

Comment:
```suggestion
```

I don't see ADIOS1 being built in this file, so Blosc1 should probably be removed, too?

Reply:
Correct, good catch! :)